app/controllers/ems_common.rb

```diff
@@ -464,6 +464,10 @@ def button
       javascript_redirect :controller         => "cloud_volume",
                           :action             => "new",
                           :storage_manager_id => params[:id]
+    elsif params[:pressed] == "cloud_volume_snapshot_create"
+      javascript_redirect :controller => "cloud_volume",
+                          :action     => "snapshot_new",
+                          :id         => find_checked_ids_with_rbac(CloudVolume).first
```

Comment:
In the future, please note the preferred syntax for this is `find_records_with_rbac(CloudVolume, checked_or_params).first`.

(I guess you missed the deprecation warning around `find_checked_ids_with_rbac` ;).)